zng/docs/zson.md

```diff
@@ -324,6 +324,13 @@ than its previous definition though multiple definitions of the same
 type are legal (thereby allowing for concatenation of otherwise
 independent sequences).
 
+One decorator is allowed per value except for nested type-union values, which
+may additional decorators to successively refine the union type for unions vals
+that live inside other union types. This allows an already-decorated value to be
+further decorated with its union type and provides as a means to distinguish
+a union value's precise member type when it is other ambiguous as described in
```

Comment:
```suggestion
further decorated with its union type and provides a means to distinguish
a union value's precise member type when it is otherwise ambiguous as described in
```